### PR TITLE
Merged and revised the content.

### PR DESCRIPTION
**Pull request by jonathanpallant**
Now uses mdslides instead of marp.

### CODE REVIEW

File: src/what-is-rust.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+
+## The Rust Foundation
+
+> ... is an independent non-profit organization dedicated to stewarding the Rust
+> programming language, nurturing the Rust ecosystem, and supporting the set of
+> maintainers governing and developing the project.
+
+---
+
+![center width:1000](./platinum-members.png)
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]: This image seems a little blurry, maybe it needs to be a bit smaller, like 900 or 800 px

File: src/what-is-rust.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+  * Professional consulting firms
+* There will soon be a *premium* Rust compiler with a support contract
+
+## No-one is an expert overnight
+
+* Budget for some training
+* Budget for some time for the team to gain experience
+* (More on this later...)
+
+## You might need a bigger computer...
```
Comment (Mirabellensaft) [inline src/what-is-rust.md:317]: This slide overwhelms me with text, I think it makes sense to make it 2, one for the quote and one for the bullet points

File: src/what-is-rust.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+* Rust is a collaborative open-source project that prides itself on inclusion.
+* There is no "owner", nor "BDFL" 
+* It has a strong financial backing
+* It is a work in progress
+
+## Is this a community I can engage with?
+
+## A strong Code of Conduct
+
+* The Rust Project, and pretty much the whole Community, follow a [Code of
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]:
```suggestion
The Rust Project, and pretty much the whole Community, follow a [Code of
```

File: src/what-is-rust.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+  Conduct](https://www.rust-lang.org/policies/code-of-conduct)
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]:
```suggestion
  Conduct](https://www.rust-lang.org/policies/code-of-conduct):
```

File: src/what-is-rust.md [hunk 5]
```diff
@@ -0,0 +1,397 @@
...
+
+## Portable
+
+* Source code is portable across multiple architectures:
+  * x86 and Arm
+  * Power, MIPS, SPARC, ...
+
+## Rust can *import* C-compatible libraries
+
+* Want to use `zlib`, `OpenSSL`, `SomeSpecialDriverLib`? Sure!
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]:
```suggestion
Want to use `zlib`, `OpenSSL`, `SomeSpecialDriverLib`? Sure!
```

File: src/what-is-rust.md [hunk 6]
```diff
@@ -0,0 +1,397 @@
...
+
+## Fast
+
+## Safe
+
+## Productive
+
+## It's increasingly popular!
+
+* [Stack Overflow Most Loved Language](https://survey.stackoverflow.co/2022/#section-most-loved-dreaded-and-wanted-programming-scripting-and-markup-languages)
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]:
```suggestion
 [Stack Overflow Most Loved Language](https://survey.stackoverflow.co/2022/#section-most-loved-dreaded-and-wanted-programming-scripting-and-markup-languages):
```

File: src/what-is-rust.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+* Hypervisors
+* Embedded Systems
+* Libraries/plugins for applications in other languages
+
+</div>
+</div>
+
+## Front-end or Back-end?
+
+* It's applicable at *every point* in the stack
```
Comment (Mirabellensaft) [inline src/what-is-rust.md]:
```suggestion
It's applicable at *every point* in the stack!
```

File: src/where-next.md [hunk 8]
```diff
@@ -0,0 +1,486 @@
...
+
+* Where the compiler cannot verify the rules are upheld, you can tell it you've
+  done the checks manually
+* We create `unsafe { }` blocks
+* Lets you access raw pointers (e.g. for memory-mapped I/O)
+* When you audit/review the code, you pay close attention to these parts!
+
+## You can describe parts of your API as `unsafe`
+
+* You are requiring the user to pay attention and do some manual checks
```
Comment (Mirabellensaft) [inline src/where-next.md]:
```suggestion
You are requiring the user to pay attention and do some manual checks
```

File: src/where-next.md [hunk 9]
```diff
@@ -0,0 +1,486 @@
...
+
+## Rust doesn't interact well with C++ code
+
+* Rust doesn't understand classes or templates
+* Neither Rust nor C++ have a stable ABI
+* Projects do exist to auto-generate bindings, like [cxx](https://crates.io/crates/cxx)
+
+## Touching the hardware requires `unsafe`
+
+* Hardware is a blob of shared mutable state and you have to manually verify
```
Comment (Mirabellensaft) [inline src/where-next.md]:
```suggestion
Hardware is a blob of shared mutable state and you have to manually verify
```

File: src/where-next.md [hunk 10]
```diff
@@ -0,0 +1,486 @@
...
+    your access to it is correct
+
+##  What you have works just fine
+
+* If it's safe enough, maintainable enough and fast enough, then you should keep it!
```
Comment (Mirabellensaft) [inline src/where-next.md]:
```suggestion
If it's safe enough, maintainable enough and fast enough, then you should keep it!
```

File: src/where-next.md [hunk 11]
```diff
@@ -0,0 +1,486 @@
...
+
+## It's early days for building critical-systems in Rust
+
+* Ferrocene is good, but C and Ada have a multi-decade head start
```
Comment (Mirabellensaft) [inline src/where-next.md]:
```suggestion
 Ferrocene is good, but C and Ada have a multi-decade head start
```

**Review by Mirabellensaft — CHANGES_REQUESTED** [review]

File: src/what-is-rust.md (hunk 1 above)
Comment (jonathanpallant) [reply src/what-is-rust.md]: I'll take out the forced scaling, but that's pretty much how they are on the Foundation website: https://foundation.rust-lang.org/members/

**Review by Mirabellensaft — APPROVED** [review]

**Pull request merged**